packages/web-react/src/types/alert.ts

```diff
@@ -14,9 +12,11 @@ export interface AriaAlertElementTypeProps<T extends ElementType = 'div'> {
 
 export interface AlertProps extends ChildrenProps, StyleProps, TransferProps {}
 
-export interface SpiritAlertProps<T extends ElementType = 'div'> extends AriaAlertElementTypeProps<T>, AlertProps {
+export interface SpiritAlertProps<T extends ElementType = 'div', S = void>
```

Comment:
I suggest using `E` for `EmotionColor`. `S` was meant for `Size`. Please change everywhere.